server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/controllers/EarthquakeController.java

```diff
@@ -1021,7 +1023,7 @@ public List<Earthquake> findEarthquakes(
     public Earthquake deleteEarthquake(@ApiParam(value = "Earthquake Id", required = true) @PathParam("earthquake-id") String earthquakeId) {
         Earthquake eq = getEarthquake(earthquakeId);
 
-        if (authorizer.canUserDeleteMember(this.username, earthquakeId, spaceRepository.getAllSpaces())) {
+        if (this.username.equals(eq.getOwner())) {
```

Comment:
For the future: can this logic be factored out somewhere in a more generic controller?

Reply:
I thought so in the beginning, but it is just very simple one line comparison with all the vales are already set. So it might be mire complicated to factor out.